packages/shared/src/components/comments/SubComment.tsx

```diff
@@ -42,13 +44,18 @@ export default function SubComment({
             lastComment ? 'h-4' : 'bottom-0',
           )}
         />
-        <ProfileImageLink className="w-8 h-8" user={comment.author} />
+        <ProfileTooltip user={comment.author}>
+          <ProfileImageLink
+            user={comment.author}
+            picture={{ size: 'medium' }}
+          />
+        </ProfileTooltip>
       </div>
       <div className="flex flex-col flex-1 items-stretch ml-2">
         <SubCommentBox>
           <CommentAuthor postAuthorId={postAuthorId} author={comment.author} />
           <CommentPublishDate comment={comment} />
-          <div className="mt-2">
+          <div className={classNames(styles.commentBox, 'mt-2')}>
```

Comment:
What is added here that the commentBox is needed?

Reply:
It was added before this line, but since the tooltip is inside of it, the selector inside that CSS module is getting applied to the tooltip content unnecessarily. It turns the links inside the tooltip as blue since the links don't have the className `commentAuthor` and I think it is safe to say that the actual comment box starts from this point.

Reply:
It was added before this line, but since the tooltip is inside of it, the selector inside that CSS module is getting applied to the tooltip content unnecessarily. It turns the links inside the tooltip as blue since the links don't have the className `commentAuthor` and I think it is safe to say that the actual comment box starts from this point.

Long story short, this was a pre-existing class for links inside the comment box. Please advise if this should be removed now since we have the markdown.

Reply:
@rebelchris your thoughts?

Reply:
I don't think we should need this anymore as the markdown should have all the comments now.
We don't use the old slugify function anymore.

If it's being overruled we should fixed it in the markdown styling rather.

Reply:
Sounds great. Especially since the CSS module only contains this specific styling. One less file for CSS once we remove this one. Thanks for the thoughts both of you!